src/test/ui/privacy/privacy-ns1.stderr

```diff
@@ -86,7 +64,19 @@ LL | use foo2::Bar;
 LL | use foo3::Bar;
    |
 
-error: aborting due to 4 previous errors
+error[E0107]: wrong number of const arguments: expected 0, found 1
```

Comment:
This is interesting fallout. I imagine we should be able to special-case this sort of error to get back the old ones, though I'm not sure how much work that would be.

Reply:
As far as I remember, the fix may be non-trivial and error-prone, so it's better to make an issue and fix it in a separate PR.